WebServer/read_page_inf.py

```diff
@@ -0,0 +1,12 @@
+def read_page(s_path, **html_messages):
```

Comment:
`html_messages` parameter is usually named `context`, which may store information about user, forms, some messages, any other data

Reply:
Corrected